standalone/CMakeLists.txt

```diff
@@ -10,20 +14,44 @@ include(../cmake/tools.cmake)
 
 include(../cmake/CPM.cmake)
 
+# Crow needs Boost 1.64 and does not provide CPM.cmake integration itself, so we have to get Boost
+# first
+find_package(Boost 1.64 QUIET)
```

Comment:
The Dockerfile at this level does not work!
We need access to `../cmake` and `../CMakeLists.txt` while build standalone